Meet Up - JavaScript/README.md

```diff
@@ -1,15 +1,29 @@
-# chrome-extension-meetup
-Its a simple extension which can open and store all the added meeting links to join quickly.
+# Chrome-Extension-MeetUp
+Its a simple extension which store all your meeting links together in one place and in one click you can join quickly.
```

Comment:
Still, there is grammatical mistake use  Grammarly to avoid grammatical mistakes